bin/templates/project/build.gradle

```diff
@@ -31,6 +31,17 @@ repositories {
 
 apply plugin: 'com.android.application'
 
+apply from: 'dependencies.gradle'
+
+dependencies {
+	compile (name:'aemm-lib-release', ext:'aar')
+
+	compile fileTree(dir: 'libs', include: ['*.jar'], excludes:['adobeMobileLibrary-*.jar'])
+
+	// SUB-PROJECT DEPENDENCIES START
+	// SUB-PROJECT DEPENDENCIES END
+}
+
```

Comment:
The phonegap-plugin-barcodescanner plugin has an aar in it with a res/xml/preferences.xml that was overwriting ours.